src/azure-cli-core/azure/cli/core/tests/test_aaz_field.py

```diff
@@ -787,7 +786,7 @@ def test_aaz_object_with_polymorphism_support(self):
             ]
         })
 
-        self.assertTrue(AAZCommand.deserialize_output(v, client_flatten=False, secret_hidden=True) == {
+        self.assertEqual(AAZCommand.deserialize_output(v, client_flatten=False, secret_hidden=True), {
```

Comment:
This test file has many of those `assertTrue` that should be replaced with something more specific.

https://github.com/jparise/flake8-assertive

It's not yet available as part of the ruff bundle: https://github.com/astral-sh/ruff/issues/2417

Reply:
Also, this file has an odd mix of both unittest asserts and pytest assert.